docker/Dockerfile.dev

```diff
@@ -0,0 +1,99 @@
+
+ARG BASE_IMAGE=python:3.10-slim
+FROM $BASE_IMAGE as builder
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+RUN apt-get update && \
+    apt-get install -q -y --no-install-recommends \
+    build-essential \
+    gcc \
+    libsqlite3-dev \
+    unattended-upgrades && \
+    unattended-upgrade -v
+
+WORKDIR /app
+
+COPY .. .
+
+RUN python -m venv /app && \
+    . ./bin/activate && \
+    pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir .
+
+
+FROM $BASE_IMAGE
+
+ARG DEBIAN_FRONTEND
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -q -y --no-install-recommends \
+    libsqlite3-0 \
+    postgresql \
```

Comment:
I suppose this could be any postgres? Dunno if that will be a problem at some point.

Reply:
I don't think it really matters. This is a dev container, so that's up to the user to set if they need it.